Tools/gsc/templates/Dockerfile.ubuntu18.04.gscapp.template

```diff
@@ -0,0 +1,35 @@
+# Integrate Graphene into the app image
+# This file is used in a multistage docker build process, in which the previous image is named "graphene"
+FROM ${appImage}
+
+# Update any packages
+RUN apt-get update \
+    && env DEBIAN_FRONTEND=noninteractive apt-get install -y \
+        python3 \
+        python3-protobuf \
+        libprotobuf-c-dev \
+        binutils \
+        openssl
```

Comment:
I had to install below packages for Ubuntu 18.04. `python3-protobuf`  alone was not working
 `python3-pip 
  && pip3 install protobuf`

without installing `pip3 install protobuf` I got below error when running python example.
```
Traceback (most recent call last):
  File "/graphene/signer/pal-sgx-get-token", line 11, in <module>
    import aesm_pb2
  File "/graphene/signer/aesm_pb2.py", line 8, in <module>
    from google.protobuf import descriptor as _descriptor
ModuleNotFoundError: No module named 'google'
```